packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -27,6 +29,23 @@ import ContextMenu from './context-menu';
 import StylesPreview from './preview';
 import { unlock } from '../../private-apis';
 
+const { Slot: GlobalStylesScreenRootSlot } = createSlotFill(
```

Comment:
Should we be exposing the API in edit-site/src/index like we do for all the slots?